src/Function.h

```diff
@@ -59,6 +59,31 @@ class CVQualifiers;
 
 ///////////////////////////////////////////////////////////////////////////////
 
+class FunctionAttribute
+{
+public:
+	string attribute;
+	int is_attribute;
```

Comment:
The name is very misleading, should be `attribute_probability` or something like that.